lib/rules/jsx-no-target-blank.js

```diff
@@ -74,6 +74,19 @@ function getStringFromValue(value) {
       if (value.expression.type === 'TemplateLiteral') {
         return value.expression.quasis[0].value.cooked;
       }
+      const expr = value.expression;
+      if (expr.type === 'ConditionalExpression') {
+        if (expr.consequent.type === 'Literal'
+        && ((expr.consequent.value && expr.consequent.value.toLowerCase() === 'noreferrer')
+        || (expr.consequent.value && expr.consequent.value.toLowerCase() === 'noopener noreferrer'))) {
```

Comment:
```suggestion
        if (
          expr.consequent.type === 'Literal'
          && (
            (expr.consequent.value && expr.consequent.value.toLowerCase() === 'noreferrer')
            || (expr.consequent.value && expr.consequent.value.toLowerCase() === 'noopener noreferrer')
          )
        ) {
```

also are you sure the consequent value will always be a string?

also, what if the value is `noreferrer noopener`?

Reply:
For `noreferrer noopener`, I can add another case. Also if the consequent's value is a string it should match one of the three cases.

Reply:
right, but if it's not a string, `.toLowerCase()` will crash

Reply:
that's why both the conditions should be true similar to in the function `attributeValuePossiblyBlank(attribute)`

Reply:
what if they're a number, or an object or array?

it's a completely acceptable answer to say "all the parsers we know about always make this a string", and maybe that was the justification for the other function - but it seems like it might be safer to have `typeof expr.consequent.value === 'string' && `?

Reply:
That makes sense! I'll update it